test/e2e/specs/writing-flow.test.js

```diff
@@ -149,13 +149,11 @@ describe( 'adding blocks', () => {
 		await pressWithModifier( META_KEY, 'b' );
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 
-		// When returning to Visual mode, backspace in selected block should
-		// reset to an unmodified default block.
+		// Reset.
```

Comment:
Concise. (The old comment seemed better. 🤔)

Reply:
the old comment was wrong (outdated). we don't switch modes

Reply:
It can probably just be removed then. Or expanded. "Remove text to reset the block." or something. 🤷‍♂️